serviceWorker.js

```diff
@@ -0,0 +1,31 @@
+const cacheName = 'fretty.cache.0.02';
+
+self.addEventListener('activate', event => {
+    event.waitUntil(
+        caches.keys().then(cacheNames => {
+            return Promise.all(
+                cacheNames.map(cache => {
+                    if(cache !== cacheName) {
+                        return caches.delete(cache);
+                    }
+                })
+            )
+        })
+    );
+});
+self.addEventListener('fetch', event => {
+    if(!event.request.url.startsWith('http')){
+        event.respondWith(
+            fetch(event.request)
```

Comment:
Should this fetch have an await?

Reply:
Maybe? Dunno. Still trying to figure all of this out.